skimage/restoration/_nl_means_denoising.pyx

```diff
@@ -12,7 +12,7 @@ ctypedef np.float64_t IMGDTYPE
 cdef double DISTANCE_CUTOFF = 5.0
 
 cdef extern from "fast_exp.h":
-    inline double fast_exp(double y) nogil
+    double fast_exp(double y) nogil
```

Comment:
Does this change bring an overhead?

Reply:
I do not think so, since it is declared inline in the header `fast_exp.h`. The warning indicated this need not be duplicated, as best I understood.